lab/src/main/kotlin/materialui/lab/components/treeView/treeView.kt

```diff
@@ -0,0 +1,38 @@
+package materialui.lab.components.treeView
+
+import kotlinx.html.DIV
+import kotlinx.html.Tag
+import kotlinx.html.TagConsumer
+import materialui.components.StandardProps
+import materialui.components.toolbar.enums.ToolbarStyle
+import org.w3c.dom.events.Event
+import react.RBuilder
+import react.RClass
+import react.ReactElement
+
+@JsModule("@material-ui/lab/TreeView")
+private external val treeViewModule: dynamic
+
+external interface TreeViewProps : StandardProps {
+    var defaultCollapseIcon: ReactElement?
+    var defaultEndIcon: ReactElement?
+    var defaultExpandIcon: ReactElement?
+    var defaultParentIcon: ReactElement?
+    var expanded: Array<String>?
+    var defaultExpanded: Array<String>?
+    var onNodeToggle: (event: Event, nodeIds: Array<String>) -> Unit?
+}
+
+@Suppress("UnsafeCastFromDynamic")
+private val treeViewComponent: RClass<TreeViewProps> = treeViewModule.default
+
+fun RBuilder.treeView(
+    vararg classMap: Pair<ToolbarStyle, String>,
+    block: TreeViewElementBuilder<DIV>.() -> Unit
```

Comment:
The component of TreeView is `ul`. So, please specify UL instead of DIV.

https://github.com/mui-org/material-ui/blob/024e176b9d2d0dc96826b0fec630823254372683/packages/material-ui-lab/src/TreeView/TreeView.js#L546-L552